python/ctsm/test/test_sys_mesh_modifier.py

```diff
@@ -72,7 +68,7 @@ def setUp(self):
         except subprocess.CalledProcessError as e:
             sys.exit(f"{e} ERROR using {configure_cmd}")
 
-    def createScripGridAndMask(self):
+    def createScripGridAndMask(self, fsurdat_in):
```

Comment:
`pylint` didn't like something in `self` being defined outside the initialization function. That was the genesis of these complaints:
```
ctsm/test/test_sys_mesh_modifier.py:150:8: W0201: Attribute 'fsurdat_in' defined outside __init__ (attribute-defined-outside-init)
ctsm/test/test_sys_mesh_modifier.py:188:8: W0201: Attribute 'fsurdat_in' defined outside __init__ (attribute-defined-outside-init)
```

Reply:
Ahhh OK. A solution could be to set it inside of __init__ to None, or to pass it in. I think I slightly prefer having it defined in __init__, you could then have some standard checking to make sure it's set to something besides None. Any thoughts there?

Reply:
I sort of like having it outside `self` here, since it's a variable that differs depending on which test you're running. Requiring it to be specified in the test also avoids the need to check that it's not `None`, because people won't be able to accidentally not specify it.

Reply:
That makes sense. OK, let's leave it as it is.